src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,14 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables) {
+        StringBuilder sb = new StringBuilder();
+        for (Printable p: printables) {
+            sb.append(p.getPrintableString() + " ");
```

Comment:
Good job using `StringBuilder`.
Its possible to simpify this line to `sb.append(p.getPrintableString()).append(" ");`